sql.txt

```diff
@@ -1,14 +1,25 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers;
+SELECT Country FROM Customers ORDER BY Country;
+SELECT CustomerName, ContactName, Country FROM Customers LIMIT 10 OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "A%" ; 
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity >"10"; 
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO products VALUES (78,"bread",2,2,"4 boxes",9.99);
+INSERT INTO products VALUES (79,"peanuts",4,9,"14 boxes",16.99);
+INSERT INTO products VALUES (80,"milk",1,2,"3 boxes",5.99);
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
+UPDATE Products SET SupplierID =0 , CategoryID=1 WHERE price> 1.50;
```

Comment:
This would set SupplierID to 0 and CategoryID to 1 where price is greater than 1.50, and would do that for every row, not the top 2. To limit the changes to the top 2, you could use TOP or LIMIT. To increase values using sql, use SET. For example, Set Price = Price +1.50 adds 1.50 to the price. 

So one answer could be 
UPDATE Products
SET Price = Price + 1.50
LIMIT 2;

or you could use the TOP clause to get a similar output.

Reply:
I was wandering if it is possible to shorten this code into 1 line? is this okay to merge?

Reply:
Hi Jessica, 
Instead of using Limit, you could use the Top clause. So:

UPDATE TOP (2) Products
SET Price = Price + 1.50;

Great job going back to your work and redoing it with a correct solution! Glad you put in that effort, and your solution is right! However, if you had a thousand rows, for example, you would need to manually retype the productID=1..2...3...1000.